src/components/ProtectedRoute/ProtectedRoute.js

```diff
@@ -20,7 +13,7 @@ const ProtectedRoute = ({ children, requiredAuthLevel }) => {
     };
 
     if (requiredAuthLevel === "anonymous") {
-        if (!userAuthenticated) {
+        if (userAuthenticated) {
             return <Navigate to="/login" />;
```

Comment:
Instead of navigating to /login can you navigate to the home route here? I'm going to approve this for now. After that one line change feel free to merge.